src/unix/uclibc/arm/mod.rs

```diff
@@ -611,6 +618,358 @@ pub const _SC_V6_LP64_OFF64: ::c_int = 0xb2;
 pub const _SC_V6_LPBIG_OFFBIG: ::c_int = 0xb3;
 pub const _SC_XOPEN_STREAMS: ::c_int = 0xf6;
 
+// Syscall table is copied from src/unix/notbsd/linux/musl/b32/arm.rs
+pub const SYS_restart_syscall: ::c_long = 0;
```

Comment:
These are Linux syscalls. Are the uclibc targets only intended for use on Linux? @alexcrichton @malbarbo ? If so, these should be in a `notbsd/linux/uclibc` folder probably, and not here.

Reply:
AFAIK uclibc has no official support nor ever been built in rust-lang/rust. These targets I think were just contributed by community porters, I don't know how accurate and/or the status of them are.